src/ContainerFactory.php

```diff
@@ -85,8 +85,11 @@ public static function build(array $config, $rebuild = false): ContainerInterfac
                 return new EmailsController($config);
             });
 
-            $container[EventCommentsController::class] = $container->factory(function ($c) use ($config) {
-                return new EventCommentsController($config);
+            $container[EventCommentsController::class] = $container->factory(static function ($container) use ($config): EventCommentsController {
```

Comment:
These closures can be `static`, we can use `$container` instead of `$c` as a parameter name, and we can add a return type declaration.